src/components/Me/Header/Logo.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Logo = () => {
```

Comment:
I don't think this is necessary, CRA already comes with a loader to import the svg directly into a component.

```
import Logo from './logo.svg'
```

Reply:
👆